### PR TITLE
documentation(deployments page) CFY-4398

### PR DESCRIPTION
**Pull request by kfurmann**

### CODE REVIEW

File: content/manager_webui/deployments-page.md [hunk 1]
```diff
@@ -12,11 +12,34 @@ relationships_link: dsl-spec-relationships.html
 ---
 {{% gsSummary %}}{{% /gsSummary %}}
 
-When clicking on the `Deployments` tab and choosing a deployment you will be able to choose one of the following:
+# Deployments Index
 
-# Topology
+A table showing all existing deployments. It shows ID, blueprint, creation and update dates of each deployment and also provides actions dropdown (see below).
```
Comment (nir0s) [inline content/manager_webui/deployments-page.md]: all existing -> all of the existing...

File: content/manager_webui/deployments-page.md [hunk 2]
```diff
@@ -12,11 +12,34 @@ relationships_link: dsl-spec-relationships.html
...
+![deployments index]({{< img "ui/ui-deployments-index.jpg" >}})
+
+When clicking on a deployment ID in the Deployments Index table, you'll be redirected to single deployment view.
```
Comment (nir0s) [inline content/manager_webui/deployments-page.md]: single deployment -> a single deployment's

File: content/manager_webui/deployments-page.md [hunk 3]
```diff
@@ -12,11 +12,34 @@ relationships_link: dsl-spec-relationships.html
...
+# Deployments Index
 
-# Topology
+A table showing all existing deployments. It shows ID, blueprint, creation and update dates of each deployment and also provides actions dropdown (see below).
+![deployments index]({{< img "ui/ui-deployments-index.jpg" >}})
+
+When clicking on a deployment ID in the Deployments Index table, you'll be redirected to single deployment view. 
+
+# Deployment View
+Here you can view deployment's details and perform actions on it.
```
Comment (nir0s) [inline content/manager_webui/deployments-page.md]: the deployment's
on it -> on the deployment

File: content/manager_webui/deployments-page.md [hunk 4]
```diff
@@ -12,11 +12,34 @@ relationships_link: dsl-spec-relationships.html
...
+
+## Actions dropdown
+Use this component to perform actions on the deployment.  
+![deployment actions]({{< img "ui/ui-deployment-actions.jpg">}})
+
+### Execute Workflow
+Pick this option to execute a worklow on the deployment. A dialog will appear with available workflows listed. To execute, pick one and click 'Confirm'.
```
Comment (nir0s) [inline content/manager_webui/deployments-page.md]: Pick this options -> Use this

File: content/manager_webui/deployments-page.md [hunk 5]
```diff
@@ -12,11 +12,34 @@ relationships_link: dsl-spec-relationships.html
...
+![Execution dialog]({{< img "ui/ui-deployment-execution-dialog.jpg">}})
+
+### Delete
+Pick this option to delete the deployment. A confirmation dialog will appear. To delete the deployment click 'Yes'.
```
Comment (nir0s) [inline content/manager_webui/deployments-page.md]: Pick this option -> Use this

File: content/manager_webui/deployments-page.md [hunk 6]
```diff
@@ -12,11 +12,34 @@ relationships_link: dsl-spec-relationships.html
...
+![Delete dialog]({{< img "ui/ui-deployment-delete-dialog.jpg">}})
+
+## Tabs
+Following tabs contain various deployment's information:
```
Comment (nir0s) [inline content/manager_webui/deployments-page.md]: The following
deployment's information -> details about the different deployments

**Comment by nir0s** [comment]
please fix conflicts

**Comment by nir0s** [comment]
?

**Comment by kfurmann** [comment]
Accidentally pulled and merged master in, fixed that, pulled and merged 3.3.0-build. should be fine.

**Pull request merged**